SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -20,22 +20,20 @@
 // created (https://github.com/intel/llvm/issues/5077) and the TEST_HALF macros
 // must be enabled when it is resolved.
 
-#include "../common.hpp"
+#include "common.hpp"
 
 using namespace cl::sycl;
```

Comment:
```suggestion
```
Not sure it's a good idea. What if we have name clash in `esimd_test` and `sycl`? Anyway, let's use the `sycl` namespace to align with SYCL-2020, not the `cl::sycl` one

Reply:
I've applied this comment in this commit: https://github.com/intel/llvm-test-suite/pull/614/commits/98e64557d3ac8bb9d222967a32688856bb78cc2d

Reply:
this statement was deleted in this commit: ce4735d0c5c845a2eb948b1c59e84651f724961e